keylime/tenant.py

```diff
@@ -743,36 +748,39 @@ def do_cvdelete(self, verifier_check=True):
         do_cvdelete = RequestsClient(self.verifier_base_url, True, tls_context=self.tls_context)
         response = do_cvdelete.delete(f"/v{self.api_version}/agents/{self.agent_uuid}", timeout=self.request_timeout)
 
-        response = response.json()
+        response_json = Tenant._jsonify_response(response, True)
```

Comment:
You want this printed here?

Reply:
This should probably throw an exception if conversion fails, so `(..., False, True)` ? A bit tricky this function now...

Reply:
Yes, it should throw an exception, as it was before